Client/Frontend/UIConstants.swift

```diff
@@ -53,3 +53,13 @@ public struct UIConstants {
     /// JPEG compression quality for persisted screenshots. Must be between 0-1.
     static let ScreenshotQuality: Float = 0.3
 }
+
+private struct TempStrings {
+    // Bug 1189902 - Offer crash reporting on first run after a crash
+    let crashReportTitle = NSLocalizedString("Oops! Firefox crashed", comment: "Title for prompt displayed to user after the app crashes")
+    let crashReportDescription = NSLocalizedString("Send a crash report so Mozilla can fix the problem?", comment: "Message displayed after the app crashes asking to send crash reports")
```

Comment:
Since the comment is already associated with the string, it doesn't need to explain what's clear from the string itself, while it's useful to explain the context: e.g. "Message displayed in the crash dialog above the buttons used to select when sending reports" or similar.